river/Output.zig

```diff
@@ -56,19 +67,23 @@ usable_box: Box,
 views: ViewStack(View) = .{},
 
 /// The double-buffered state of the output.
-current: State = State{ .tags = 1 << 0 },
-pending: State = State{ .tags = 1 << 0 },
+current: State = State{ .tags = 1 << 0, .layout = false },
+pending: State = State{ .tags = 1 << 0, .layout = false },
```

Comment:
Using a default value here would be more idiomatic. (should probably have done that for `tags`)
```suggestion
pending: State = State{ .tags = 1 << 0, .layout = false },
```